src/lib/query/organizations/UserByEmail.js

```diff
@@ -0,0 +1,25 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query getUserByEmail($email: String!){
```

Comment:
There is  query called `userByEmailAndOrganization` that already exists, but it only has the `groupRoles` field on it, so you won't be able to return the projects that are associated to this user with this query, but I think this is fine because users are allocated to groups, not projects 
```
query ubm {
  userByEmailAndOrganization(email:"testuser@example.com", organization: 1){
    email
    groupRoles{
      name
    }
  }
}
```